app/src/main/kotlin/com/adriangl/pokeapi_mvvm/utils/mini/FluxUtil.kt

```diff
@@ -0,0 +1,131 @@
+package com.adriangl.pokeapi_mvvm.utils.mini
```

Comment:
I might move this to mini/rx package, since this implementation is coupled to Rx. I mean, this would potenitally be extracted to a `mini-rx` package

Reply:
86560256